### DISCUSSION TITLE
Create a Mastodon account

### DISCUSSION THREAD

**Discussion by orblivion** [bostonopen/bostonopen.github.io#6] [Ideas]
Mastodon is getting generally popular. FOSS cohort is gonna be heavily represented. We could have a social presence by following it with our personal accounts, re-posting meeting announcements to get it to a wider audience.

Probably fosstodon. Using one of the Boston-based ones could be interesting but that's probably less FOSS people.

**Comment by pdurbin** [comment]
@orblivion sure! If you want to go ahead and create a Mastodon account and be in charge of it, please go ahead! I'd only suggest that you give the password to at least one other person to reduce the bus number. 😄

**Comment by orblivion** [comment]
In person we mentioned fosstodon.org, but it's invite-only. Any issues with floss.social? That's what the Sandstorm community ended up choosing.

**Comment by pdurbin** [comment]
floss.social looks good to me!

**Comment by orblivion** [comment]
I put in some account basics, lmk if you have ideas for improvements: https://floss.social/@bostonopen

I'll link from Mastodon to our site after we add a link in the other direction: https://github.com/bostonopen/bostonopen.github.io/pull/7 The PR explains why.

**Comment by pdurbin** [comment]
Merged! Looks great! Thanks!

<img width="907" alt="Screenshot 2023-09-13 at 6 39 27 PM" src="https://github.com/bostonopen/bostonopen.github.io/assets/21006/3b90cecf-4052-4505-a0ad-2fe76add2d60">

**Comment by orblivion** [comment]
And now we're verified

![Screenshot 2023-09-13 at 18-50-47 Boston Open Dev (@bostonopen@floss social)](https://github.com/bostonopen/bostonopen.github.io/assets/41976/83be29d4-405a-401f-a56f-41dc329f3d95)

Only remaining things:

* On my browser at least, for whatever reason floss.social doesn't always have their images load. Though I guess if you're following from another instance it won't be a problem?
* Next month, let's remember to ask people to follow!
